app/src/main/res/layout/item_speaker.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="?android:attr/listPreferredItemHeight"
+  android:background="?selectableItemBackground"
+  android:orientation="horizontal">
+
+  <ImageView
+    android:id="@+id/speaker_image"
+    android:layout_width="wrap_content"
+    android:layout_height="match_parent"
+    android:scaleType="centerInside"
+    tools:src="@mipmap/ic_launcher" />
+
+  <TextView
+    android:id="@+id/speaker_name"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_gravity="center"
+    android:textColor="@android:color/primary_text_light"
```

Comment:
Use the theme attribute if possible: `?android:textColorPrimary` (all non-`layout_` attributes belong in a style, really)